src/services/response-service.ts

```diff
@@ -1,4 +1,7 @@
 import * as crypto from 'crypto'
+// @ts-ignore - TS hates JS without types
+// why does this default export function need to be imported like this?
```

Comment:
We can probably remove the comment since the answer is weird stuff that TS does 🤷‍♂ https://stackoverflow.com/a/55055982